nims-notes-things/src/App.js

```diff
@@ -1,9 +1,14 @@
+import React from 'react';
+// import pencil from './penLogo.png';
 import "./App.css";
 import Form from "./Form";
 
 function App() {
   return (
     <div className="App">
```

Comment:
You can also wrap you app in a theme https://styled-components.com/docs/advanced